toonz/sources/common/tcore/tstopwatch.cpp

```diff
@@ -352,9 +352,9 @@ TUINT32 TStopWatch::getSystemTime()
 TStopWatch::operator string()
 {
 	char buffer[256];
-	ostrstream out(buffer, sizeof(buffer));
+	ostringstream out(buffer);
 	out << m_name.c_str() << ": " << (int)getTotalTime() << " u" << (int)getUserTime() << " s" << (TINT32)getSystemTime();
-	return string(buffer, out.pcount());
+	return string(buffer);
```

Comment:
This is wrong. While `ostrstream` used `buffer` as the storage for the string, `ostringstream` always uses an internal buffer which can be accessed through `out.str()` and makes a copy of the string given to it. Therefore, when `ostringstream` is constructed, it will make a copy some uninitialized garbage and probably crash the program.